ssl/quic/quic_record_tx.c

```diff
@@ -837,7 +837,9 @@ int ossl_qtx_write_pkt(OSSL_QTX *qtx, const OSSL_QTX_PKT *pkt)
          * Ensure TXE has at least MDPL bytes allocated. This should only be
          * possible if the MDPL has increased.
          */
-        if (!qtx_reserve_txe(qtx, NULL, txe, qtx->mdpl))
+        TXE_LIST *txl = &qtx->free;
```

Comment:
The change makes sense. Would it be possible to move declaration of `txl` either to the beginning of `for()` loop or event the `ossl_qtx_write_pkt()`. thanks a lot.